hbp_knowledge/proteostasis/Brekk2018.bel

```diff
@@ -73,123 +73,149 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
+UNSET Confidence
 
-SET Evidence = " Furthermore, LAMP2A acts as the rate-limiting step for the activity of the CMA pathway, with rate of the assembly / disassembly of the LAMP2A complex in the lysosomal membrane being paramount for its function (Cuervo and Dice 1996;Bandyopadhyay et al. 2008)."
 
+SET Evidence = " Furthermore, LAMP2A acts as the rate-limiting step for the activity of the CMA pathway, with rate of the assembly / disassembly of the LAMP2A complex in the lysosomal membrane being paramount for its function (Cuervo and Dice 1996;Bandyopadhyay et al. 2008)."
+SET Confidence = "Low"
 p(HGNC:LAMP2) reg bp(GO:"chaperone-mediated autophagy")
+#Can you let me know which statement shows about the next two Statements
 p(HGNC:LAMP2,loc(GO:cytoplasm)) -| bp(GO:"chaperone-mediated autophagy")
 p(HGNC:LAMP2,loc(GO:"lysosomal membrane")) -> bp(GO:"chaperone-mediated autophagy")
+UNSET Confidence
 
 SET Evidence = "In vivo evidence shows that age-related decline in CMA function coincides with reduced levels of LAMP2A (Cuervo and Dice 2000b), and that restoration of LAMP2A abundance in the aged liver is sufficient to rescue CMA function in aged animals (Zhang and Cuervo 2008). Although it is maximally activated upon prolonged starvation (Cuervo et al. 1995), CMA activity has been also found specifically upregulated in response to oxidative stress (Hubbi et al. 2013; Kiffin 2004), genotoxic insult (Park et al. 2015), and - in a compensatory fashion - upon impaired macroautophagy (Koga et al. 2011a)"
-
+SET Confidence = "High"
 bp(GO:aging) neg p(HGNC:LAMP2)
 bp(GO:aging) -| bp(GO:"chaperone-mediated autophagy")
 path(MESH:Starvation) -> bp(GO:"chaperone-mediated autophagy")
 bp(GO:"cellular response to oxidative stress") -> bp(GO:"chaperone-mediated autophagy")
 bp(GO:macroautophagy) neg bp(GO:"chaperone-mediated autophagy")
+UNSET Confidence
 
 SET Evidence = " Importantly, CMA dysfunction has been linked to a variety of neurodegenerative conditions, most notably Parkinson’s Disease (PD), as well as to the ageing process, as part of the generalized decline of the activity of the protein degradation systems (Xilouri and Stefanis 2016; Xilouri and Stefanis 2015). "
-
+SET Confidence = "High"
 bp(GO:"chaperone-mediated autophagy") -- path(MESH:"Neurodegenerative Diseases")
 bp(GO:"chaperone-mediated autophagy") -- path(DOID:"Parkinson's disease")
 bp(GO:"chaperone-mediated autophagy") -- bp(GO:aging)
 bp(GO:"chaperone-mediated autophagy") isA bp(GO:"cellular protein catabolic process")
+UNSET Confidence
 
 SET Evidence = "a recent in vivo study identified significant changes to large portions of the proteome upon LAMP2A knock-down and consequent CMA dysfunction in the aging liver (Schneider et al. 2014)."
-
+SET Confidence = "Medium"
 SET Anatomy = "liver"
 p(HGNC:LAMP2) reg a(MESH:Proteome)
-bp(GO:"chaperone-mediated autophagy") reg a(MESH:Proteome)
+a(MESH:Proteome) reg bp(GO:"chaperone-mediated autophagy")
```

Comment:
CAM is responsible for turnover of proteins so I went with the previous statement